src/plugins/embeddable/public/plugin.tsx

```diff
@@ -70,6 +82,19 @@ export interface EmbeddableStart {
     embeddableFactoryId: string
   ) => EmbeddableFactory<I, O, E> | undefined;
   getEmbeddableFactories: () => IterableIterator<EmbeddableFactory>;
+
+  /**
+   * Given {@link ChartActionContext} returns a list of `data` plugin {@link Filter} entries.
+   */
+  filtersFromContext: (context: ChartActionContext) => Promise<Filter[]>;
```

Comment:
What do you think of instead of making new plugin dependency `embeddable -> data`, have this high-level function on data.actions instead. 

As context it would get `unknown`